app/src/views/features/api-client/client-view/APIClientView.tsx

```diff
@@ -293,7 +293,7 @@ const APIClientView: React.FC<Props> = ({ apiEntry, notifyApiRequestFinished })
           className="api-client-body"
           direction="vertical"
           cursor="row-resize"
-          sizes={[40, 60]}
+          sizes={entry.request.contentType === RequestContentType.JSON ? [60, 40] : [40, 60]}
```

Comment:
After adding the custom toolbar in editor component, the code editor textarea height shrinked even more. Changed the ratio so that by default there' enough space to write the request body